cheroot/test/test_conn.py

```diff
@@ -8,8 +8,12 @@
 import six
 import pytest
 
-from cheroot._compat import HTTPConnection, HTTPSConnection, NotConnected, BadStatusLine
-from cheroot._compat import ntob, urlopen
+from cheroot._compat import HTTPConnection
```

Comment:
Regarding 79 chars limitation. Nowadays people usually set higher threshold, as we don't have 80 char terminals limitation today. Still it's a good practice to avoid long lines to make code better readable.